mmedit/models/editors/gfpgan/gfpgan_generator.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+
+from mmedit.registry import MODULES
+from .gfpgan_modules import ResBlock, StyleGAN2GeneratorCSFT
+
+
+@MODULES.register_module()
+class GFPGANv1Clean(nn.Module):
+    """The GFPGAN architecture: Unet + StyleGAN2 decoder with SFT.
```

Comment:
We can add a link to the original implementation of this module and provide the full name of `SFT`.